src/utils/domains.ts

```diff
@@ -1,10 +1,21 @@
+import { Id } from '@colony/colony-js';
 import { Colony } from '~types';
 import { notNull } from './arrays';
 
-export const findDomain = (domainId?: string | null, colony?: Colony) => {
+export const findDomain = (
+  domainId: number | null | undefined,
+  colony: Colony | undefined,
+  skipRootDomain?: boolean,
+) => {
   const domains = colony?.domains?.items;
   if (!domains || !domainId) {
     return undefined;
   }
-  return domains.filter(notNull).find(({ id }) => id === domainId);
+  return domains
+    .filter(notNull)
+    .find(({ id }) =>
+      skipRootDomain && (domainId === undefined || domainId === Id.RootDomain)
+        ? Number(id) !== Id.RootDomain
+        : Number(id) === domainId,
```

Comment:
This logic is actually wrong - it should be comparing the `domainId` against `nativeId`.